core/include/optee_msg_supplicant.h

```diff
@@ -48,14 +48,8 @@
  */
 
 /*
- * SQLite file system access
- */
-#define OPTEE_MSG_RPC_CMD_SQL_FS	8
```

Comment:
Reserving still makes sense, this number shouldn't be reused in the nearest future. How about:
```
/* Was OPTEE_MSG_RPC_CMD_SQL_FS, which isn't supported any longer */
#define OPTEE_MSG_RPC_CMD_SQL_FS_RESERVED	8
```